wingtips-spring-boot2-webflux/src/main/java/com/nike/wingtips/springboot2/webflux/WingtipsSpringBoot2WebfluxConfiguration.java

```diff
@@ -113,7 +114,7 @@ public WingtipsSpringBoot2WebfluxConfiguration(WingtipsSpringBoot2WebfluxPropert
      */
     @Bean
     @ConditionalOnWebApplication(type = ConditionalOnWebApplication.Type.REACTIVE)
-    public WingtipsSpringWebfluxWebFilter wingtipsSpringWebfluxWebFilter() {
+    public WebFilter wingtipsSpringWebfluxWebFilter() {
```

Comment:
SB 2.6 introduced a breaking change where this ended up being a circular dependency error (see https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.6-Release-Notes#circular-references-prohibited-by-default). Changing the return type to `WebFilter` fixes that error without changing the functionality of what we're trying to do.